src/graphics/images/dumptga.cpp

```diff
@@ -116,4 +116,26 @@ void dumpTGA(const Common::UString &fileName, const ImageDecoder *image) {
 	file->flush();
 }
 
+void dumpTGA(Common::WriteStream &stream, const ImageDecoder *image) {
```

Comment:
That function duplicates an awful lot of code. :/

I'm opening the actual file pretty late in the original `dumpTGA()`, to decrease the possibility of leaving a blank file when an early check throws, but that's not really essential. I'd rather have a function that writes to a stream, and a small wrapper function that opens a file and then calls that other function.